src/m365/spo/commands/field/field-set.ts

```diff
@@ -195,12 +205,16 @@ class SpoFieldSetCommand extends SpoCommand {
       return `${args.options.listId} in option listId is not a valid GUID`;
     }
 
-    if (!args.options.id && !args.options.name) {
-      return `Specify id or name`;
+    if (!args.options.id && !args.options.fieldTitle && !args.options.name) {
+      return `Specify id or fieldTitle`;
+    }
+
+    if (args.options.name && args.options.fieldTitle) {
+      return `Specify only fieldTitle. Option name is deprecated.`;
     }
 
-    if (args.options.id && args.options.name) {
-      return `Specify viewId or viewTitle but not both`;
+    if (args.options.id && (args.options.fieldTitle || args.options.name)) {
+      return `Specify id or fieldTitle but not both`;
     }
```

Comment:
do you think we could refactor to option set and along the way do the same for listId/listTitle options